udata/core/dataset/api.py

```diff
@@ -192,7 +192,13 @@ def parse_filters(datasets, args):
             except Topic.DoesNotExist:
                 pass
             else:
-                datasets = datasets.filter(id__in=[d.id for d in topic.datasets])
+                datasets = datasets.filter(
```

Comment:
maybe add this to all elements supported by topics, not only datasets(or maybe in another PR but we can track an issue)

Reply:
My goal here was to stay isofunctionnal, i.e handling Datasets and Reuses in a Topic. The Reuse API does have a `topic=` filter, but it does not refer to Topics, it's a different concept 😬 Cf `REUSE_TOPICS`.